src/app/modules/main/module.nim

```diff
@@ -66,6 +67,8 @@ type
     communitiesModule: communities_module.AccessInterface
     appSearchModule: app_search_module.AccessInterface
     nodeSectionModule: node_section_module.AccessInterface
+    allChatsModel: chat_model.Model
```

Comment:
@osmaczko `allChatsModel` and  `allChatsModelVariant` should be part of the `view` class, cause `view` is our adapter for interaction with the UI (qml). So please update that accordingly.